zeus/device/gpu.py

```diff
@@ -507,8 +510,10 @@ def setPowerManagementLimit(self, value: int) -> None:
     @_handle_amdsmi_errors
     def resetPowerManagementLimit(self) -> None:
         """Resets the power management limit for the specified GPU to the default value."""
-        info = amdsmi.amdsmi_get_power_cap_info(self.handle)
-        amdsmi.amdsmi_set_power_cap(self.handle, 0, cap=info.default_power_cap)
+        info = amdsmi.amdsmi_get_power_cap_info(self.handle)  # Returns in W
+        amdsmi.amdsmi_set_power_cap(
+            self.handle, 0, cap=int(info["default_power_cap"] * 1e6)
+        )  # expects value in microwatts
```

Comment:
<img width="1146" alt="Screenshot 2024-05-01 at 8 21 27 PM" src="https://github.com/ml-energy/zeus/assets/29395896/0743ab77-676f-4a7f-b4d5-ccd2c82fd1fe">
Holy shit... 5.7 returns in Watts but 6.0 returns in microwatts.... Is everything based on 5.7? Then let's keep it and later figure out how to make it work for 6.0...

Reply:
<img width="723" alt="image" src="https://github.com/ml-energy/zeus/assets/68670266/237b017e-77d3-4d62-94df-99df721a3db1">

This is what I see, I was going off the source code for 6.0. So it should be correct for ROCM 6.0. 

As for pytorch, it looks like it just got full support for ROCM 6.0 with the release of [Pytorch 2.3](https://pytorch.org/blog/pytorch2-3/) a week ago.

Should we stick to ROCM 6.0 then?

Reply:
Yep, if everything is consistent with 6.0, let's keep it that way!